check_python.sh

```diff
@@ -142,8 +143,9 @@ pushd "${PROJECT_REPO}" > /dev/null || exit 1
         fi
       fi
     else
-      if grep '^#!/bin/.*python' "${script_file}"; then
-        if grep '^#!/bin/.*python3' "${script_file}"; then
+      if (head -1 "$script_file" | grep -q -E '^#!(/usr)?/bin/.*python'); then
```

Comment:
Why not let `file` do the work here for us?

```suggestion
         if $(file "$script_file") | grep -e 'Python script'; then
```

Reply:
I thought we ran all our python linting as python3 anyway, even though much of the code is python2.